osbenchmark/metrics.py

```diff
@@ -1966,6 +1986,7 @@ def single_latency(self, task, operation_type, metric_name="latency"):
 class GlobalStats:
     def __init__(self, d=None):
         self.op_metrics = self.v(d, "op_metrics", default=[])
+        self.kpi_metrics = self.v(d, "kpi_metrics", default=[])
```

Comment:
What is the scope for kpi metrics? Aside from recall and maybe ndcg, what else would go in here?

Reply:
I think any performance metric for k-NN is in scope for this. Right now it seems like recall is the most major one but if k-NN adds ndcg we could also track that in OSB. (The process for adding new metrics would be to calculate/add it to the metadata in the search query runner and then pull it from the metadata into this `kpi_metrics` list). Likewise if we or the anomaly detection plugin want to track false positives rate/specificity.

I mainly wanted to explicitly distinguish between algorithm metrics like recall and `op_metrics` like latency.

Reply:
Oh I see. KPI though might include latency right? It seems like a good category here would be `correctness_metrics` or `relevance_metrics`